pkg/cmd/infra/deployer/deployer.go

```diff
@@ -136,7 +136,7 @@ func getDeployerContext(controllerGetter replicationControllerGetter, namespace,
 	glog.Infof("Inspecting %d potential prior deployments", len(allControllers.Items))
 	for i, controller := range allControllers.Items {
 		if oldName := deployutil.DeploymentConfigNameFor(&controller); oldName != newConfig.Name {
-			glog.Infof("Disregarding deployment %s (doesn't match target deploymentConfig %s)", controller.Name, oldName)
+			glog.Infof("Disregarding deployment %s (doesn't match target deployment configuration %s)", controller.Name, oldName)
```

Comment:
Don't think it would make sense to put the namespace here. It is mentioned in the Infof above and in this case it would be a dup. since the namespace will need to be mentioned twice.